_websiteutils/make_redirects_links.py

```diff
@@ -67,18 +66,38 @@ def findlast(fname, tocheck):
     return None
 
 
-html_redirect = """<!DOCTYPE HTML>
-<html lang="en">
-    <head>
-        <meta charset="utf-8">
-        <meta http-equiv="refresh" content="0;url={newurl}" />
-        <link rel="canonical" href="https://matplotlib.org/{canonical}" />
-    </head>
-    <body>
+html_redirect = """
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
+  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```

Comment:
Use html5 doctype, no need for xhtml at this point.

Reply:
thats just from the top of matplotlib.org?
```
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```

Reply:
Not in the current theme: https://github.com/matplotlib/mpl-brochure-site/blob/gh-pages/index.html